src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.laptop {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  margin-top: $cards-horisontal_shift;
+  width: 5 * $cards-width + 3 * $cards-horisontal_shift;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $cards-width;
+  padding: 32px 16px 16px ;
+  margin-left: 50px;
+  margin-bottom: $cards-horisontal_shift;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 0;
+  }
+
+  &__img {
+    margin: 0 auto;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin: 40px 0 0;
```

Comment:
let's use only margin-bottom

if you need to reset margins for some elements, you can reset it for tags